ramalama/model.py

```diff
@@ -1,6 +1,20 @@
 import os
 import sys
-from ramalama.common import container_manager, exec_cmd, default_image
+from ramalama.common import container_manager, exec_cmd, default_image, in_container
+
+
+file_not_found = """\
+RamaLama requires the "%s" command to be installed on the host when running with --nocontainer.
```

Comment:
I like that we just said %s command is to be installed (and not suggested a particular package manager). I think in similar python related messages we suggested apt, dnf, pip, pipx (and that's leaving out other popular ones like zypper)

Better to let the user decide what's best, maybe that's `make; make install` for them, who knows